models/public/mobilenet-yolo-syg/mobilenet-yolo-syg.md

```diff
@@ -0,0 +1,124 @@
+# mobilemet-yolo-syg
+
+## Use Case and High-Level Description
+
+  This is a Keras\* version of `mobilenet-yolov4` model designed to perform real-time vehicle detection
+  on FFmpeg-OpenVINO. It can realize illegal lane change detection and speed detection by developing FFmpeg.
+  The weights are pretrained by BDD100k and retrained by our own dataset.
+  For details see repository <https://github.com/ermubuzhiming/OMZ-files-download/>,
```

Comment:
The repository you mention actually does not provide more details on the model. It does not contain model training pipeline or any description of how you actually got your .PB file and how one may fine-tune the model on their specific data.